torch/_dynamo/symbolic_convert.py

```diff
@@ -2304,7 +2304,14 @@ def inline_call_(
         if code.co_name in ("__setitem__", "__setattr__") and not (
             args is not None
             and len(args) > 0
-            and isinstance(args[0], variables.CustomizedDictVariable)
+            and isinstance(
+                args[0],
+                (
+                    variables.CustomizedDictVariable,
+                    variables.NNModuleVariable,
+                    variables.dicts.HFPretrainedConfigVariable,
```

Comment:
Curious: why is this so special? There are a bunch of others in `variables.dicts` as well...

Reply:
Looks like a sentinel for this could be `constant_attributes_tracker_EXPORT_ONLY`. In general I'd like these to share some kind of protocol instead of listing them manually here.

Reply:
I actually don't think this is needed.